react/components/molecules/tabs/tabs.js

```diff
@@ -26,35 +26,29 @@ export class Tabs extends PureComponent {
         };
     }
 
-    onPressTab = (tag, index) => {
-        this.props.navigation.navigate(tag, index);
+    onPressTab = route => {
+        this.props.navigation.navigate(route);
     };
 
-    _isSelected = index => {
-        const {
-            navigation: {
-                state: { index: selected }
-            }
-        } = this.props;
-        return selected === index;
+    _isSelected = id => {
+        return this.props.state.routeNames[this.props.state.index] === id;
```

Comment:
Please confirm that the `tabs` story still runs without any issues (console errors, etc).